release-please-config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bump-minor-pre-major": true,
+    "separate-pull-requests": true,
+    "pull-request-title-pattern": "chore: release ${component} ${version}",
+    "pull-request-header": ":package: Staging a new release",
+    "packages": {
+        ".": {
+            "package-name": "sdk",
```

Comment:
Not sure about **_sdk_** as package name, but can't come up with any better

Reply:
Well if have nothing else, that is what we used before.